packages/cli/fragments/fragment-leptos/src/app.rs

```diff
@@ -1,4 +1,4 @@
-use leptos::web_sys::{Event, SubmitEvent};
+use web_sys::{Event, SubmitEvent};
```

Comment:
```suggestion
use leptos::leptos_dom::ev::{InputEvent, SubmitEvent};
```
and update the `update_name` closure to take `InputEvent` instead

Reply:
Apparently `on:input` event type is `Event` and I get an error when using `InputEvent`...

Reply:
Seems like leptos v0.2 is missing a re-export of `Event` type, Remove the `Event` import all together and change the on:input handler to 
```rs
on:input=move |ev| {
  let v = event_target_value(&ev);
  update_name(v)
}
```
then update `update_name` function to take `String` type instead

Reply:
Why not just remove the type from the `update_name` closure?

Reply:
if it works, yes but last time I created the template it wasn't able to infer the type.

Reply:
Works for me.

Reply:
cool, lets go with that then